audio/munt-mt32emu/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                munt-mt32emu
+version             2.4.0
+revision            0
+
+homepage            http://munt.sourceforge.net/
+description         Munt library
+long_description    mt32emu is a C/C++ library which allows to emulate (approximately) the Roland MT-32, CM-32L and LAPC-I synthesiser modules.
+
+categories          audio emulators
+platforms           darwin
+license             LGPL-2.1
+
+maintainers         nomaintainer
+
+master_sites        sourceforge:project/munt/munt/${version}
+
+distname            munt-${version}
+
+checksums           rmd160  9f3c76daf60294613e4e2646eba9e76d506c85ff \
+                    sha256  b4f7054df1d3f89e2cc683ff6182c4d0a272daceffc4d27fd968b6eaebcdc9ed \
+                    size    474789
+
+configure.args-append \
+                    -Dmunt_WITH_MT32EMU_SMF2WAV=FALSE \
+                    -Dmunt_WITH_MT32EMU_QT=FALSE
```

Comment:
Looking good! Please also address the lint warning by adding a newline at the end.